APMrover2/mode_auto.cpp

```diff
@@ -131,6 +131,40 @@ float ModeAuto::get_distance_to_destination() const
     return 0.0f;
 }
 
+// get desired location
+bool ModeAuto::get_desired_location(Location& destination)
+{
+    switch (_submode) {
+    case Auto_WP:
+        if (g2.wp_nav.is_destination_valid()) {
+            destination = g2.wp_nav.get_destination();
+            return true;
+        } else {
```

Comment:
Remove else and break, just return false.

Reply:
yup, removed else and break.